app/views/devise/mailer/invitation_instructions.html.erb

```diff
@@ -1,6 +1,6 @@
 <p><%= t("devise.mailer.invitation_instructions.hello", email: @resource.email) %></p>
 
-<p><%= t("devise.mailer.invitation_instructions.someone_invited_you", url: root_url) %></p>
+<p><%= t("devise.mailer.invitation_instructions.description", url: root_url) %></p>
```

Comment:
This is minor, but it might be preferable to continue using the `someone_invited_you` translation key because it's an override of a key coming from the gem `devise_invitable`.